include/xbps.h.in

```diff
@@ -207,6 +207,13 @@
  */
 #define XBPS_FLAG_UNPACK_ONLY 		0x00001000
 
+/**
+ * @def XBPS_FLAG_DOWNLOAD_ONLY
+ * Do not configure packages in the transaction, just unpack them.
+ * Must be set through the xbps_handle::flags member.
+ */
+#define XBPS_FLAG_DOWNLOAD_ONLY 	0x00003000
```

Comment:
That flag value implies XBPS_FLAG_UNPACK_ONLY, why do you check both?

Reply:
Poorly thought out reasoning.